### PR TITLE
Bugfix/typos Fixes bad wording in language files

### PR DESCRIPTION
**Pull request by juan-cortes**
### 📝 Description

Found a small typo while developing something else, decided to do a run down of our language files and found a few more.

### ❓ Context

- **Impacted projects**: `ledger-live-desktop, ledger-live-mobile` 
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/LIVE-3374` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo
No demo

### 🚀 Expectations to reach
Only wording

### CODE REVIEW

File: apps/ledger-live-desktop/static/i18n/en/app.json [hunk 1]
```diff
@@ -1283,7 +1283,7 @@
       "noSearchResults": "Sorry, we did not find any search results.",
       "retrySearchKeyword": "Please retry the search with another keyword.",
       "retrySearchParams": "Please retry the search with another parameters.",
-      "trackFavAssets": "Track your favourite",
+      "trackFavAssets": "Track your favorite",
```
Comment (hedi-edelbloute) [inline apps/ledger-live-desktop/static/i18n/en/app.json:1286]: I think both are fine :D American vs british english

File: .changeset/sweet-clocks-shave.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": minor
```
Comment (haammar-ledger) [inline .changeset/sweet-clocks-shave.md:2]: Super unimportant but I think these changes are more of a `patch`

**Review by hedi-edelbloute — APPROVED** [review]

File: apps/ledger-live-desktop/static/i18n/en/app.json (hunk 1 above)
Comment (juan-cortes) [reply apps/ledger-live-desktop/static/i18n/en/app.json:1286]: We settled for american for all

File: .changeset/sweet-clocks-shave.md (hunk 2 above)
Comment (juan-cortes) [reply .changeset/sweet-clocks-shave.md:2]: :ah: you're probably right

**Review by chabroA — APPROVED** [review]

**Review by ofreyssinet-ledger — APPROVED** [review]
noice! good catches 👍

**Review by elbywan — APPROVED** [review]
because 3 approvals is not enough 👌

**Pull request merged**